platform-configs/msi-pro-z690-a-wifi-ddr4.robot

```diff
@@ -55,10 +55,10 @@ ${DEVICE_WINDOWS_USERNAME}=                         user
 ${DEVICE_WINDOWS_PASSWORD}=                         windows
 ${DEVICE_WINDOWS_USER_PROMPT}=                      PS C:\\Users\\user>
 
-${DEVICE_UBUNTU_USERNAME}=                          ${UBUNTU_USERNAME}
-${DEVICE_UBUNTU_PASSWORD}=                          ${UBUNTU_PASSWORD}
-${DEVICE_UBUNTU_USER_PROMPT}=                       ${UBUNTU_USER_PROMPT}
-${DEVICE_UBUNTU_ROOT_PROMPT}=                       ${UBUNTU_ROOT_PROMPT}
+${DEVICE_UBUNTU_USERNAME}=                          ---
+${DEVICE_UBUNTU_PASSWORD}=                          ---
+${DEVICE_UBUNTU_USER_PROMPT}=                       ---
+${DEVICE_UBUNTU_ROOT_PROMPT}=                       ---
```

Comment:
Why do we change it? Can't it be left as it was before using the UBUNTU_ variables?

Reply:
@miczyg1 sorry, change removed